ikasaneip/test/src/main/java/org/ikasan/testharness/flow/database/DatabaseHelper.java

```diff
@@ -0,0 +1,63 @@
+package org.ikasan.testharness.flow.database;
+
+
+import org.apache.commons.lang3.ArrayUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.util.StopWatch;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class DatabaseHelper {
```

Comment:
Can we wire this class into IkasanFlowTestRule ?

rather then having to wire the DatabaseHelper into your test it would be nice to call

flowTestRule.clearIkasanDBState()

Reply:
Not too keen to use ikasanFlowTestRule to be honest. IkasanFlowTestRule isnt a bean so not readily easy to wire things in such as xa data sources. Think we can come up with a better framework that does wire services in. These helper classes are just to get the ball rolling they will become part of the framework. Prefer this approach to a big bang.

Reply:
Also ideally all clean up will be done automatically by the test framework without the user having to call this themselves. (Although they could configure this)